### PR TITLE
Update icon bug for instagram and snapchat

### PR DESCRIPTION
**Pull request by Ajmal**
- Fix size and color inconsistency
- add 'icon' class

### Before
Before

### After
Before

### CODE REVIEW

**Comment by Ajmal** [comment]
signed CLA

File: src/snippets/icon-snapchat.liquid [hunk 1]
```diff
@@ -1 +1,20 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 512.016 512.016" style="enable-background:new 0 0 512.016 512.016;" class="icon">
```
Comment (huguestennier) [inline src/snippets/icon-snapchat.liquid]: Can you remove the added `style` attribute and remove the multiple lines?

Comment (Ajmal) [reply src/snippets/icon-snapchat.liquid]: @huguestennier Fixed

**Comment by huguestennier** [comment]
There's some issues with other icons but I will open a new PR.

Thanks for your work!

**Pull request merged**